cfgov/unprocessed/css/enhancements/forms.less

```diff
@@ -300,6 +300,80 @@ legend {
 }
 
 /* topdoc
+  name: Input with button bar
```

Comment:
I can definitely add it -- if I don't get it in this sprint, I should have time in the next one.